Automattic/jetpack
# Launchpad: Show modal to return to Launchpad after the user saves a change in the site editor

#### Summary
Currently, there's no easy way for the user to go back to Launchpad after they have made changes in the site editor. This PR creates a Modal that opens every time the user saves a change.

#### Testing instructions:
There are two possibilities for testing this - testing on the local jetpack docker setup or testing it on WPCOM by using `jetpack rsync`

##### Notes:
You should already have setup your local jetpack environment. I've [followed these instructions](https://github.com/Automattic/jetpack/blob/trunk/tools/docker/README.md#quick-install-wordpress) for the docker setup

##### Testing on WPCOM
1. Create a new Link in Bio site
2. Checkout this branch
3. Run `jetpack watch` and select `plugins -> jetpack`
4. Run `jetpack rsync` to sync your local changes to your sandbox ([Fieldguide page](PCYsg-eg0-p2))
5. Connect to your sandbox. You should see your changes applied.
6. Sandbox `public-api.wordpress.com` and your site URL
7. Go to `YOUR_SITE/wp-admin`
8. Go to the site editor (Appearance -> Editor)
9. Make a change in the site editor and click "Save"
10. You should see the Launchpad modal appear
11. Clicking "Back to Edit" or the close button should close the modal
12. Clicking "Launch Site" should take you to the Launchpad screen

**NOTE:** You might have to include `define( 'JETPACK_BETA_BLOCKS', true );` in your sandbox `wp-config.php`. This has been flaky for me, initially it didn't work without it but now it does so I'm not sure if this is required for an extension plugin...

https://user-images.githubusercontent.com/20927667/204922136-2b052594-66c8-40d4-919b-ee95d9edf106.mov





Fixes #69757

##### Other information:


##### Jetpack product discussion



##### Does this pull request change what data or activity we track or use?
